documentation/smg/11_ISIS_release.md

```diff
@@ -1,79 +1,161 @@
-# ISIS Release Process
+# SMG-11: ISIS Release Process
 
-Horace is deployed separately to a number of systems across ISIS
+Horace is deployed separately to a number of systems across ISIS. These are:
 
-## ISIS Compute
+- IDAaaS: This is currently the normal access point for users of Horace in ISIS.
+- SCARF: This is an alternative HPC environment. In contrast to the deployment on IDAaaS, the current SCARF deployment is experimental, and its parallel capabilities are more limited. 
+Details of this are given below, including contacts for improving the installation.
+- ISISCOMPUTE: This is the base location for code deployed to IDAaaS, and can also be used directly by experienced users only. The section on this below gives reference details
+for use of this system. 
```

Comment:
Where it have come from. ISISCOMPUTE is dead. 
All mentions to it should be removed

Reply:
Apologies, my text appears to be based on a clone on 20 Sept, before your changes - don't know how this happened. I will update based on the more recent versions.